api/Port.go

```diff
@@ -151,3 +194,168 @@ func (vrsConnection *VRSConnection) UpdatePortMetadata(name string, metadata map
 
 	return nil
 }
+
+// Disconnected will retry connecting to OVSDB
+// and continue to register for OVSDB updates
+func (ovsdbc OvsdbClient) Disconnected(ovsClient *libovsdb.OvsdbClient) {
+}
+
+// Locked is a placeholder function
+func (ovsdbc OvsdbClient) Locked([]interface{}) {
+}
+
+// Stolen is a placeholder function
+func (ovsdbc OvsdbClient) Stolen([]interface{}) {
+}
+
+// Echo is a placeholder function
+func (ovsdbc OvsdbClient) Echo([]interface{}) {
+}
+
+// Update will provide updates on OVSDB table updates
+func (ovsdbc OvsdbClient) Update(context interface{}, tableUpdates libovsdb.TableUpdates) {
+    ovsdbc.mUpdateChan <- &tableUpdates
+}
+
+// GetNuagePortTableUpdate will register with OVSDB
+// for Nuage Port table updates and return as soon as
+// port table entry gets populated
+func (ovsdbc *OvsdbClient) GetNuagePortTableUpdate() <-chan ClientEvent {
+    var err error
+    var conn *libovsdb.OvsdbClient
+    ovsdbc.mUpdateChan = make(chan *libovsdb.TableUpdates)
+    clientChan := make(chan ClientEvent, 1)
+    for {
```

Comment:
Get rid of this. This method needs to be called on the VRSConnection object. Please re-use the ovsdb connection in that object.